components/search/TierToggler.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <button
+        class="toggle-button"
+        data-qa="tier toggler"
+        @click="toggleHandler"
+      >
+        {{ !toggle ? button.show : button.hide }}
+      </button>.
+    </p>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'TierToggler',
+
+    props: {
+      toggled: {
```

Comment:
I think toggled is a little confusing since semantically either state could be toggled.
Have it be `activeState`